### ISSUE TITLE
want to use python api for bloomberg - have terminal access. Trouble using many different wrappers 

### ISSUE THREAD

**Issue by hodumAL** [matthewgilbert/pdblp#55]
#### Code Sample, a copy-pastable example if possible

```python
import pdblp
con = pdblp.BCon(debug=False, port=8194)
con.start()

```
#### Problem description
pdblp.pdblp:WARNING:Message Received:
SessionStartupFailure = {
    reason = {
        source = "Session"
        category = "IO_ERROR"
        errorCode = 9
        description = "Connection failed"
    }
}

#### Expected Output
Want to be able to just get historical data, time series data etc. but first trying to establish connection.
#### Version Information
I am using desktop version of the API, and have bloomberg terminal on my desktop. Any help would be appreciated

**Comment by matthewgilbert** [comment]
Have you logged into the terminal? If so and it is still not working you may need to start the COM, e.g.
```
start C:\blp\DAPI\bbcomm.exe
```
Note that the location of `bbcomm.exe` may be different on your machine.

**Comment by matthewgilbert** [comment]
I believe you need to install the actual terminal, available from https://www.bloomberg.com/professional/support/software-updates/, to get access to the API. I am unsure about this, but do not think you can use it via Bloomberg Anywhere. That being said, you could get a definitive answer from Bloomberg help via `HELP HELP` in the terminal.

**Comment by matthewgilbert** [comment]
Yes I've come across this before and it can be annoying, particularly because the (stable) Python API hosted at https://www.bloomberg.com/professional/support/api-library/ is only compatible with the experimental C/C++ SDK. I believe they provide compatible versions if you instead download them from `WAPI` within the terminal, however this is one reason to not worry about it and just use the `conda` package for managing `blpapi` and `pdblp` available on `conda-forge`, e.g.
```
conda install -c conda-forge pdblp
```
